apps/cli/src/adapters/node/repository.ts

```diff
@@ -20,6 +20,21 @@ const findRoot = (dir: string): Result<string, Error> => {
   return findRoot(parent);
 };
 
+const existsTurboConfig = (repoRoot: string): Result<boolean, Error> => {
+  const fileName = "turbo.json";
+  const turboPath = join(repoRoot, fileName);
+  if (existsSync(turboPath)) {
+    return ok(true);
+  }
+
+  return err(
+    new Error(
+      `${fileName} not found in repository root. Make sure to have Turbo configured for the monorepo.`,
+    ),
+  );
+};
```

Comment:
I would also check if `turbo` is present in the root `packageJson` file, and it is the latest supported version (`2.x`, you can use the semver package to check that constraint)